mhctools/netmhc_pan.py

```diff
@@ -58,5 +59,13 @@ def NetMHCpan(
             process_limit=process_limit,
             extra_flags=extra_flags)
 
+    elif "NetMHCpan version 4.0" in output_str:
```

Comment:
Nit, but maybe merge this and the other 2 sections into:

```
version_class = None
if "NetMHCpan version 2.8" in output_str:
    version_class = NetMHCpan28
elif "NetMHCpan version 3.0" in output_str:
    version_class = NetMHCpan3
...
else:
    raise ...

return version_class(...
```

Reply:
Changed it to something similar but a little less clever

Reply:
Cool, even better

Reply:
Whoops, I've revealed my secret identity